config/environments/production.rb

```diff
@@ -90,12 +90,15 @@
     if ScihistDigicoll::Env.lookup(:smtp_host).nil?
       raise RuntimeError, "Please specify smtp_host in local_env.py so we can send emails."
     end
+
     config.action_mailer.delivery_method = :smtp
     config.action_mailer.smtp_settings = {
-      address: ScihistDigicoll::Env.lookup(:smtp_host),
-      domain: 'sciencehistory.org',
-      ssl: false,
-      enable_starttls_auto: false
+      :address => "email-smtp.us-east-1.amazonaws.com",
```

Comment:
Don't hard-code this address in, use it from `ScihistDigicoll::Env`, there is even an `stmp_host` key already there. 

oops sorry just noticed this was marked "wip", I"m reviewing prematurely sorry!

Reply:
Yup -- but I have fixed this. I also got rid of the vestigial `email_server_ip_dns` setting in both Ansible and scihist_digicoll.